### PR TITLE
Issue/auth refactored fixes

### LINKED ISSUE #291: Small fixes after authorization was refactored
* [x] add blueprint for `zaken:create-status` to prefent 5000 error in zaak-detail pages of old ZAC 
* [x] fix data migration of the old permissions to the new model
* [x] create command to add blueprint permissions for all zaaktypen to make it easy for administrators.

### PR DESCRIPTION
**Pull request by annashamray**
fixes #291

### CODE REVIEW

File: backend/src/zac/accounts/management/commands/add_blueprint_permissions_for_zaaktypen.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            name
+            for name, perm in registry.items()
+            if isinstance(perm.blueprint_class(), ZaakTypeBlueprint)
+        ]
+        added = []
+        for zaaktype in zaaktypen:
+            for permission in permissions:
+                obj, created = BlueprintPermission.objects.get_or_create(
+                    permission=permission,
+                    policy__zaktype_omschrijving=zaaktype.omschrijving,
```
Comment (damm89) [inline backend/src/zac/accounts/management/commands/add_blueprint_permissions_for_zaaktypen.py:31]:
```suggestion
                    policy__zaaktype_omschrijving=zaaktype.omschrijving,
```

File: backend/src/zac/accounts/management/commands/add_blueprint_permissions_for_zaaktypen.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                    policy__catalogus=zaaktype.catalogus,
+                    defaults={
+                        "policy": {
+                            "catalogus": zaaktype.catalogus,
+                            "zaktype_omschrijving": zaaktype.omschrijving,
```
Comment (damm89) [inline backend/src/zac/accounts/management/commands/add_blueprint_permissions_for_zaaktypen.py:36]:
```suggestion
                            "zaaktype_omschrijving": zaaktype.omschrijving,
```

**Review by damm89 — CHANGES_REQUESTED** [review]

File: backend/src/zac/accounts/migrations/0025_migrate_to_permission_definitions.py [hunk 3]
```diff
@@ -28,24 +29,27 @@ def migrate_to_permission_definitions(apps, _):
...
+                if permission_name != zaken_download_documents.name:
+                    zaak_permission_definition = PermissionDefinition.objects.create(
+                        object_type=PermissionObjectType.zaak,
+                        permission=permission_name,
+                        policy={
+                            "catalogus": permission_set.catalogus,
+                            "zaaktype_omschrijving": zaaktype_omschrijving,
+                            "max_va": permission_set.max_va,
+                        },
+                    )
```
Comment (SilviaAmAm) [inline backend/src/zac/accounts/migrations/0025_migrate_to_permission_definitions.py:42]: Shouldn't line 44-45 also be indented to be in the if statement?
```
for auth_profile in permission_set.authorizationprofile_set.all():
    auth_profile.permission_definitions.add(zaak_permission_definition)
```

Comment (annashamray) [reply backend/src/zac/accounts/migrations/0025_migrate_to_permission_definitions.py:42]: You are right! Thanks for cathing it!

**Review by damm89 — APPROVED** [review]

**Pull request merged**